### DISCUSSION TITLE
Lazer not reading my beatmap imports

### DISCUSSION THREAD

**Discussion by Lorphex** [ppy/osu#15672] [Q&A] [answered]
After installing and reinstalling Lazer several times, I finally got it to import maps to the correct location. After nearly 4 hours of importing maps, I messed with the tablet settings and hit play>solo. The game crashed. I tried playing again and now it says there are no beatmaps. I checked the install location and it is 138 GB, so the beatmaps were imported. But they can't be read. I tried restarting the game but no dice.
.log files are attached.

[network.log](https://github.com/ppy/osu/files/7561426/network.log)
[performance.log](https://github.com/ppy/osu/files/7561427/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7561428/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7561431/updater.log)
[database.log](https://github.com/ppy/osu/files/7561424/database.log)
.

**Comment by peppy** [comment, marked as answer]
Are you able to attach the logs from a fresh run? The ones you attached are from the import run, which don't show any crashes.

**Reply by Lorphex** [reply]
Those logs were pulled after the restart. I just launched the game again but logs didn't refresh. Is there anything specific I need to do in lazer to get you the logs?

**Reply by smoogipoo** [reply]
Did you try to migrate the game (via settings)? That's the only reason I can think of that the logs aren't refreshing. Have you tried opening the osu! folder via the button in settings instead?

**Reply by Lorphex** [reply]
Yes I used the in-game migration tool. When I open the osu! folder from in-game it takes me to AppData>Roaming>osu which is where I changed the game file from.

**Reply by Lorphex** [reply]
I have an osu-lazer folder on a different drive that is 138 GB from all the songs that were imported.

**Reply by smoogipoo** [reply]
> When I open the osu! folder from in-game it takes me to AppData>Roaming>osu which is where I changed the game file from.

Sounds like something went wrong in the migration. You probably want to open `AppData/Roaming/osu/storage.ini` and make sure that it points to the new location.
`FullPath = C:\Path\To\Other\Location\Here`

**Reply by Lorphex** [reply]
I don't have a file called storage.ini in %APPDATA%/osu
![image](https://user-images.githubusercontent.com/59071051/143841004-c414cd2a-b071-425a-8371-cff494107d7a.png)

**Reply by bdach** [reply]
which is probably the issue. please create it and point it at your custom file location as described above.

**Reply by Lorphex** [reply]
I don't know if just creating storage.ini would've worked, mostly because I had no idea if there was more text that could've been in that file. So I did something else that I admittedly should've tried first. Osu didn't redirect at all, since all of the folders that should've gone to osu-lazer are still there. I renamed osu-lazer in the custom path, and then redirected osu, so a new folder would be created. I then deleted the new osu-lazer folder, and renamed the original one back to osu-lazer. Now everything, including songs, seem to be redirecting properly.

**Reply by bdach** [reply]
the single line would be enough, yes. either way good to hear that your issue is resolved now
